apps/explorer/components/Block/index.tsx

```diff
@@ -11,13 +12,15 @@ import { routes } from '../../config/routes'
 import { EntityHeading } from '../EntityHeading'
 import { ContentLayout } from '../ContentLayout'
 import { ExplorerBlock } from '@siafoundation/explored-types'
+import { ArrowLeft16, ArrowRight16 } from '@carbon/icons-react'
```

Comment:
Update this import to `@siafoundation/react-icons` (this library centralizes all our icon usage to one package, and also fixes some issues with @types/icons-react when consuming our libraries from npm). I also always auto-import from @carbon directly - maybe there is a way we can make vs code ignore it.